File: personnage.cpp
#include "personnage.h"

/**
 * @file personnage.cpp
 * Fichier cpp pour la classe Personnage
 */

/**
 * @brief Constructeur
 * @param x : entier contenant la coordonnee x initiale du personnage
 * @param y : entier contenant la coordonnee y initiale du personnage
 */
Personnage::Personnage(int x, int y):ObjetMobile(x,y){
    this->setImg(":/images/Joueur/playerDown.png");
}

File: src/sokoban_code/Personnage.java
package sokoban_code;

/**
 * Classe representant un personnage
 * 
 * @version 1.0
 * @see ObjetMobile
 */
public class Personnage extends ObjetMobile {
	// Constructeurs
	/**
	 * Constructeur par defaut
	 */
	public Personnage() {
	}

	/**
	 * Construit l'objet personnage selon les coordonnees x et y
	 * 
	 * @param x entier contenant la coordonnee x de l'objet
	 * @param y entier contenant la coordonnee y de l'objet
	 */
	public Personnage(int x, int y) {
		super(x, y);
		setImg(":/images/Joueur/playerDown.png");
	}
}
